### ISSUE TITLE
Liked returns no results

### ISSUE THREAD

**Issue by a2hill** [mopidy/mopidy-soundcloud#37]
Running the SC extension on Mopidy 0.19.4 with the Moped 0.4.0 frontend. When I click the 'Liked' button in the extension I see the spinning wheel indicating that it is loading. While this is happening I get messages in the console such as

    INFO     'Too Close feat. Detour City' can't be streamed from SoundCloud
    INFO     'Wilkinson - Heartbeat ft. P Money & Arlissa' can't be streamed from SoundCloud
    INFO     'Wilkinson - Heatwave ft K. Flay [PREVIEW]' can't be streamed from SoundCloud
    INFO     'Kangaroo Court' can't be streamed from SoundCloud
    INFO     'Moiez vs Daphne & Project 46 - Crime (3LAU Mix)' can't be streamed from SoundCloud
    INFO     'Devil's Work (Dirty South Remix)' can't be streamed from SoundCloud

After which the loading wheel stops running and no results are returned.

**Comment by Mebus** [comment]
It's the same problem here:

<code>
2015-05-15 17:43:34,256 INFO [2069:SoundCloudBackend-3] mopidy_soundcloud.soundcloud: 'Nils Hoffmann - Balloons (Club Mix)' can't be streamed from SoundCloud
2015-05-15 17:43:34,273 INFO [2069:SoundCloudBackend-3] mopidy_soundcloud.soundcloud: 'Chase and Status - Let You Go (Nero Remix)' can't be streamed from SoundCloud
2015-05-15 17:44:45,302 INFO [2069:SoundCloudBackend-3] mopidy_soundcloud.soundcloud: 'Dark Horse Feat. Juicy J' can't be streamed from SoundCloud
</code>
and so forth.

**Comment by devonbarrett** [comment]
Bump, still experiencing this.

```
ERROR    SoundCloudBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/site-packages/mopidy/core/library.py", line 112, in _browse
    result = backend.library.browse(uri).get()
  File "/usr/lib/python2.7/site-packages/pykka/future.py", line 299, in get
    exec('raise exc_info[0], exc_info[1], exc_info[2]')
  File "/usr/lib/python2.7/site-packages/pykka/actor.py", line 200, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/site-packages/pykka/actor.py", line 294, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/site-packages/mopidy_soundcloud/library.py", line 166, in browse
    return self.list_liked()
  File "/usr/lib/python2.7/site-packages/mopidy_soundcloud/library.py", line 78, in list_liked
    logger.debug('Adding liked track %s to vfs' % data.name)
AttributeError: 'list' object has no attribute 'name'
```

**Comment by ZenithDK** [comment]
Please don't "bump" issues - the maintainers will get around to looking at the issue when they have time, not based on the number of people bumping issues.
You are generating a lot more noise (and e-mail) than I think you realize.

**Comment by jodal** [comment]
The original issue here is old and the error in the last comment was fixed by #69 at least. Please open a new issue if this is still an issue.
